src/site/content/en/blog/quictransport/index.md

```diff
@@ -0,0 +1,324 @@
+---
+title: Experimenting with QuicTransport
+subhead: QuicTransport is a new API offering low-latency, bi-directional, client-server messaging. Learn more about its use cases, and how to give feedback about the future of the implementation.
+authors:
+  - jeffposnick
+description: QuicTransport is a new API offering low-latency, bi-directional, client-server messaging. Learn more about its use cases, and how to give feedback about the future of the implementation.
+date: 2020-06-03
+updated: 2020-06-03
+hero: hero.jpg
+hero_position: center
+alt: |
+  Blurry picture of traffic. Photo by Robin Pierre on Unsplash
```

Comment:
@kaycebasques Have we started adding photo credits to the hero's alt tag? It's not a bad idea if we haven't. We should be discuss this and be consistent.

Reply:
I don't think that's an appropriate use of the `alt` attribute. The attribute should describe the contents of the image for someone who can't see it. The photographer and online location of the image is metadata. cc @robdodson to confirm.

Reply:
Yeah I would be inclined to lean toward Kayce's suggestion, especially if we include the metadata elsewhere, such as at the bottom of the article.

I think the rule of thumb with alt text is that shorter is generally better. So if there's something that we can omit, especially on a mostly presentational image, then we may want to do so to improve the efficiency of the user experience.

Reply:
FWIW, I copied https://raw.githubusercontent.com/GoogleChrome/web.dev/master/src/site/content/en/blog/wakelock/index.md as a starting point for this post, and followed that example for the `alt` metadata. Happy to change it.